elements/howto-checkbox/howto-checkbox.js

```diff
@@ -30,7 +30,7 @@
     <style>
       :host {
         display: inline-block;
-        background: url('{%PATH%}/images/unchecked-checkbox.svg') no-repeat;
```

Comment:
Is this not working or why did you remove the placeholder?

Reply:
oh I did it so I could use the element in my demo.html temporarily during testing. Still not sure what would be the best way to pull the checkbox into my demo.